docs/examples/caffe_build_index.md

```diff
@@ -0,0 +1,54 @@
+The following is a concrete example of performing nearest neiborhood computation
+using a set of ten butterfly images. It assumes you have set up the
+caffe (vrc2) environment (http://caffe.berkeleyvision.org/install_apt.html)
```

Comment:
This descriptor implementation also works on master (what I'm working with at the moment). The other Caffe descriptor that uses the external c++ executable requires rc2, however.